lib/node/NodeTargetPlugin.js

```diff
@@ -9,9 +9,18 @@ const ExternalsPlugin = require("../ExternalsPlugin");
 
 /** @typedef {import("../Compiler")} Compiler */
 
-const builtins =
+let builtins;
+try {
+	builtins =
+		// eslint-disable-next-line node/no-unsupported-features/node-builtins,node/no-deprecated-api
+		require("module").builtinModules || Object.keys(process.binding("natives"));
```

Comment:
Why double `Object.keys(process.binding("natives"))`?

Reply:
It's a fallback in case builtinModules returns null without throwing an exception. This line left unchanged from original version of code